cmd/e2e/e2e.go

```diff
@@ -63,5 +68,22 @@ func main() {
 		glog.Error("Invalid --times (negative or no testing requested)!")
 		os.Exit(1)
 	}
+
+	if context.Provider == "aws" {
+		awsConfig := "[Global]\n"
+		if cloudConfig.Zone == "" {
+			glog.Error("gce_zone must be specified for AWS")
```

Comment:
I'm fine with not renaming in this PR, but please file a bug on this.  We just need to figure out the ripple of renaming them.